package.json

```diff
@@ -30,12 +30,12 @@
   },
   "devDependencies": {
     "grunt": "~0.4.0",
-    "it": "~1.0.1",
+    "it": "~1.1.0",
     "grunt-contrib-jshint": "~0.2.0",
     "grunt-contrib-nodeunit": "~0.1.2"
   },
   "peerDependencies": {
-    "it": "~1.0.0"
+    "it": "^1.0.0"
```

Comment:
This is the main question here. I can put ~1.1.0 if you guys think thats better.

I like this though to avoid changing this more often.